### PR TITLE
Add miniforge sub team

### PR DESCRIPTION
**Pull request by isuruf**
@conda-forge/core, this needs 50% to pass

### CODE REVIEW

**Comment by hmaarrfk** [comment]
whats the mandate of this team? deploying miniforge for architectures that conda-forge supports but miniconda doesn't? i.e. aarch64?

File: src/orga/subteams.rst [hunk 1]
```diff
@@ -166,3 +166,30 @@ the following are members of the staged-recipes team and have commit rights.
...
+This sub-team is broadly responsible for developing, maintaining and releasing
+miniforge installers.
+
+Members
+-------
+In addition to [core](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/core.csv)
+and [emeritus](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/emeritus.csv),
+the following are members of the miniforge team and have commit and admin rights.
+
+- Mark Harfouche <mark.harfouche@gmail.com>
```
Comment (scopatz) [inline src/orga/subteams.rst]: Please add me to this team

File: src/orga/subteams.rst [hunk 2]
```diff
@@ -166,3 +166,30 @@ the following are members of the staged-recipes team and have commit rights.
  - Peter M. Landwehr <>
  - Patrick Sodré <psodre@gmail.com>
  - Sylvain Corlay <sylvain.corlay@gmail.com>
+
+
+
+Miniforge Sub-Team
+============
```
Comment (scopatz) [inline src/orga/subteams.rst]: This line needs to go the full way under the title

File: src/orga/subteams.rst (hunk 1 above)
Comment (scopatz) [reply src/orga/subteams.rst]: explicitly that is

**Comment by scopatz** [comment]
> deploying miniforge for architectures that conda-forge supports but miniconda doesn't? i.e. aarch64?

According to the text this is for any arch

**Comment by isuruf** [comment]
> According to the text this is for any arch

Agree. The team (which includes core) can decide what platforms to support.

File: src/orga/subteams.rst [hunk 3]
```diff
@@ -166,3 +166,30 @@ the following are members of the staged-recipes team and have commit rights.
...
+Dynamic
+
+Responsibility
+--------------
+This sub-team is broadly responsible for developing, maintaining and releasing
+miniforge installers.
+
+Members
+-------
+In addition to [core](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/core.csv)
```
Comment (CJ-Wright) [inline src/orga/subteams.rst]: Could this go in the above section? Just because core has rights doesn't make them members.

Comment (scopatz) [reply src/orga/subteams.rst]: Yeah, it seems like this was Copy-pasted from the other sections, which we should also adjust but maybe that is a different PR

Comment (isuruf) [reply src/orga/subteams.rst]: Which section?

Comment (CJ-Wright) [reply src/orga/subteams.rst]: The responsibilities section (namely the commit rights bit)

**Comment by CJ-Wright** [comment]
>The team (which includes core) 

@isuruf do you mean that this subteam includes all of core?

File: src/orga/subteams.rst (hunk 3 above)
Comment (isuruf) [reply src/orga/subteams.rst]: Sure. Can you suggest a wording or push to this branch?

**Comment by isuruf** [comment]
> @isuruf do you mean that this subteam includes all of core?

That's what I intended. Let me know if you disagree.

**Comment by CJ-Wright** [comment]
I think it would be better if compartmentalized. That way the team is more lean and the voting body is not too large. Would language similar to the auto-tick bot's team work?

`For large scale (affecting >20% of packages) this sub-team will inform and discuss with the core team about the upcoming migration prior to starting the migration.`

A clause like that would enable the team to make its own decisions on day to day operations and then bring things to core for discussion if warranted.

You could also say that members of core and emeritus have rights as [ex officio members](https://en.wikipedia.org/wiki/Ex_officio_member) where it is custom to abstain.

File: src/orga/subteams.rst (hunk 3 above)
Comment (CJ-Wright) [reply src/orga/subteams.rst]: Will do (pending discussion below on exact language)

**Comment by hadim** [comment]
The miniforge builder is in a good state now: https://github.com/conda-forge/miniforge/pull/2

Feel free to review/comment.

File: src/orga/subteams.rst (hunk 1 above)
Comment (hadim) [reply src/orga/subteams.rst]: Is that possible to be part of the subteam too?

**Comment by hadim** [comment]
I just wanted to let you know that the miniforge PR is done now. All the blocking deps have been released: https://github.com/conda-forge/miniforge/pull/2

**Comment by mbargull** [comment]
Thanks for putting this in, for the votes and thanks to the miniforge team for their future work!

**Comment by CJ-Wright** [comment]
Can someone please provision the team on github?

**Comment by isuruf** [comment]
Done. https://github.com/orgs/conda-forge/teams/miniforge

**Comment by scopatz** [comment]
Woohoo!

**Pull request merged**